bugsnag-android-core/src/androidTest/java/com/bugsnag/android/BugsnagStoreMigratorTest.kt

```diff
@@ -0,0 +1,125 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.internal.BugsnagStoreMigrator
+import org.junit.After
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import java.io.File
+import java.util.UUID
+
+class BugsnagStoreMigratorTest {
+    private var bugsnagStoreMigrator = BugsnagStoreMigrator
```

Comment:
I don't think this variable adds much, the test should rather directly reference the `BugsnagStoreMigrator` object